hw/ip/gpio/data/gpio.hjson

```diff
@@ -134,7 +134,7 @@
       swaccess: "ro",
       hwaccess: "hwo",
       tags: [// data_in is ro register, so exclude its readback check
-             "excl:CsrNonInitTests:CsrExclWriteCheck"],
+             "excl:CsrAllTests:CsrExclAll"],
```

Comment:
Why is this change needed? I think we'd probably need an explanation?

Reply:
This changes was needed because the CSR reset check function was checking also the value of the data_in input, and getting a mismatch with the model, that is no make sense to verify this input after reset. That's why is excluded.

Reply:
During the read_and_check_all_csrs_after_reset() function call, the data_in input will have some value that is mismatching with what it should be expected from reset default value setted in the hjson gpio file.

GPIO Input data read value

    Offset: 0x10
    Reset default: 0x0
    Reset mask: 0xffffffff

Reply:
Oh, sorry. I meant the comment on line 136 in that file :-)